geomap/src/components/LeafletMap.vue

```diff
@@ -56,24 +59,67 @@ import {
 import MarkerCluster from "./MarkerCluster.vue";
 import { Vue } from "vue-class-component";
 import { useStore } from "vuex";
+import L from "leaflet";
+import { Alarm, Node } from "@/types";
 import { Coordinates } from "@/types";
 
 let leafletReady = ref(false);
 let leafletObject = ref("");
 let visible = ref(false);
 let map: any = ref();
 const store = useStore();
+let iconPath;
```

Comment:
What's the reason for having this `iconPath` variable?  Doesn't seem to be used anywhere, and `setMarkerColor` could just return the string directly rather than returning the result of setting this variable, I would think.

Reply:
Yes, I removed iconPath variable.